Mathlib/RingTheory/KrullDimension/Regular.lean

```diff
@@ -120,6 +120,13 @@ lemma _root_.ringKrullDim_quotient_span_singleton_succ_eq_ringKrullDim_of_mem_ja
   exact supportDim_quotSMulTop_succ_eq_supportDim_mem_jacobson reg
     ((annihilator R R).ringJacobson_le_jacobson hx)
 
+lemma _root_.ringKrullDim_quotSMulTop_succ_eq_ringKrullDim_of_mem_jacobson {x : R}
+    (reg : IsSMulRegular R x) (hx : x ∈ Ring.jacobson R) :
+    ringKrullDim (QuotSMulTop x R) + 1 = ringKrullDim R := by
+  rw [← supportDim_quotient_eq_ringKrullDim, ← supportDim_self_eq_ringKrullDim]
+  exact supportDim_quotSMulTop_succ_eq_supportDim_mem_jacobson reg
+    ((annihilator R R).ringJacobson_le_jacobson hx)
```

Comment:
Can you move this before `ringKrullDim_quotient_span_singleton_succ_eq_ringKrullDim_of_mem_jacobson` and use this to golf that?